integration_tests/sdk/delete_workflow_test.py

```diff
@@ -0,0 +1,106 @@
+from time import sleep
+
+import pytest
+from aqueduct.error import InvalidRequestError
+from constants import SENTIMENT_SQL_QUERY
+from utils import (
+    delete_flow,
+    generate_new_flow_name,
+    get_integration_name,
+    get_response,
+    run_flow_test,
+)
+
+from aqueduct import LoadUpdateMode
+
+LIST_INTEGRATION_OBJECTS_TEMPLATE = "/api/integration/%s/objects"
```

Comment:
We have something similar to this scattered throughout the code (testing, api client). Wonder if we can create an enum for the BE routes used throughout the Python codebase.